relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -526,7 +526,7 @@ mod tests {
         };
 
         let extracted = extractor.extract(event.value().unwrap()).unwrap();
-        insta::assert_debug_snapshot!(event.value().unwrap().spans, @r#"
+        insta::assert_debug_snapshot!(event.value().unwrap().spans, @r###"
```

Comment:
Do we have to have 3 `###` here?  The new `beta` rust linter does not like it and the new release won't either and will make your CI fail, since we treat warnings as errors.  It would be good idea to keep this to the linter likings 😄 , one `#` where it's possible

Reply:
`insta` did that when it updated the snapshot. Manually reverted now.